lib/test_cases/regex/95.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Avoid using absolute css units.",
```

Comment:
The main idea behind this was around hard coding of height, not sure how much that makes sense, but this one is also good. Shall we consider that as well?

Reply:
I guess we should keep it. Because this helps us to maintain responsiveness. With height, we can have something like use height: auto instead of hardcoding. Let me add this also in next PR and in sheet.

Reply:
I'm definitely fine with this one... 👍